New Skill: Changelog watch (#159)
PostHog/context-mill: transformation-config/skills/changelog-watch/description.md

- **Bash** — to create the config directory (`mkdir -p ~/.posthog/changelog-watch`)
- **WebFetch** — to pull competitor changelog and blog pages
- **WebSearch** — fallback when WebFetch returns a client-rendered shell

## Config file location

Config is stored at **`~/.posthog/changelog-watch/config.json`** (user-level, shared across repos). On every run:

1. Try to read the config.
2. If it exists, summarise it back to the user and ask whether to run as-is or edit.
3. If it does not exist, run the first-run interview below.

Emit a `[STATUS]` line on entry so the Wizard's live spinner reflects the current sub-step:

```
[STATUS] Loading changelog-watch config
```

## Step 1 — Load or build config

### If `~/.posthog/changelog-watch/config.json` exists

Read it, then summarise to the user in one line:

> Loaded config: **{company}** — {N} categories, {M} competitors. Run as-is, or update first?

If they want to edit, ask which part (positioning / categories / competitors / focus list) and apply only that change. Show the proposed JSON diff to the user before writing.

### If the config is missing — first-run interview

> Ask these as free-form chat questions, one at a time. **Do not use `AskUserQuestion`** — this skill ships through PostHog's Wizard and MCP, and Wizard surfaces don't support structured prompts.

Reviewer: ```suggestion
Ask these as free-form chat questions, one at a time. Use the `mcp__wizard-tools__wizard_ask` tool to surface questions to the users.
```
